src/menu.rs

```diff
@@ -52,9 +52,9 @@ impl Menu {
     }
   }
 
-  pub fn add_children(&mut self, children: Menu, title: &str, enabled: bool) {
+  pub fn add_submenu(&mut self, submenu: Menu, title: &str, enabled: bool) {
```

Comment:
shouldn't this function signature follow the same as `MenuItem::Submenu` so it would be something like this `pub fn add_submenu(&mut self, title: &str, enabled: bool, submenu: Menu)`

Reply:
Perfect thx